content/guides/applications/custom-skills/setup.md

```diff
@@ -15,83 +15,80 @@ Setting up a Custom Skill is a multi-step process.
 
 ### 1. Log in to the Developer Console
 
-Head over to the [Developer Console][devconsole] and select **Create New App**.
+Log into Box and navigate to the 
+[Developer Console][devconsole]. Select **Create New App**.
 
 ### 2. Create a Custom Skill
 
-Select the **Custom Skill** option from the list of application types and select
-**Next**.
+Select **Box Custom Skill** option from the list of application types. A modal
+will appear to prompt the next step.
 
 <ImageFrame border>
 
-![Application selection screen](../images/app-types-skill.png)
+![Application selection screen](../images/select-app-type.png)
 
 </ImageFrame>
 
 ### 3. Provide a name
 
-On the next screen, provide a unique name for your application. This name needs
-to be unique across all applications on Box.
+Finally, select a unique name for your application and click **Create App**.
 
 <ImageFrame border width="600" center>
 
-![App name form](../images/app-name.png)
+![App name form](../images/skill-name.png)
 
 </ImageFrame>
 
 ## Approval
 
-Before a Custom Skill can be used it needs to be assigned to a folder for which
-the skill should trigger.
+You must select a folder that will trigger your skill before you can start using
+it. 
 
 <CTA to='g://applications/custom-skills/approval'>
   Learn more about approving Custom Skills
 </CTA>
 
 ## Basic configuration
 
-Before a Custom Skill can be enabled enabled on a folder, some basic additional
-configuration needs to be set up.
+Before a Custom Skill can be enabled enabled on a folder, you must complete
+some additional configuration.
 
 ### Invocation URL
 
-Custom Skills work by sending a payload for every file uploaded a remote URL.
-This URL is called the invocation URL.
+For every file uploaded, copied, or moved into the selected folder, your skill
+will send a payload to a remote URL. This URL is called the invocation URL.
 
 The Invocation URL can be any HTTP endpoint representing a server, development
-machine, or serverless function. The only condition is that the URL is publicly
-available and accessible by the Box servers. For this reason, `localhost` is not
-a valid address.
-
-To set up the Invocation URL, head over to the [Developer Console][devconsole],
-select your application, and select on the "Configuration" panel on the left
-hand side.
+machine, or serverless function. The only requirement is that the URL is 
+publicly available and accessible by Box servers. For this reason, `localhost` 
+is not a valid address.
 
-Scroll down to the "Invocation URL" section.
+To set up the Invocation URL, navigate to the ** Configuration** tab of the 
+[Developer Console][devconsole] and scroll down to the "Invocation URL" section.
 
 <ImageFrame border width="600" center>
 
 ![App name form](../images/app-invocation-url.png)
 
 </ImageFrame>
 
-Fill in a secure HTTPs address and save the form. The invocation URL has now
+Fill in a secure HTTPs address and save the form. The invocation URL is now
```

Comment:
Two items: Let's capitalize HTTPS for consistency (HTTPs -> HTTPS) and for the last sentence, recommend changing it to "The invocation URL is now configured." (drop **been**)